src/Illuminate/Foundation/Testing/DatabaseMigrations.php

```diff
@@ -10,9 +10,13 @@
     public function runDatabaseMigrations()
     {
         $this->artisan('migrate');
+    }
 
-        $this->beforeApplicationDestroyed(function () {
-            $this->artisan('migrate:rollback');
-        });
+    /**
+     * @after
+     */
+    public function runDatabaseMigrateRefresh()
+    {
+        $this->artisan('migrate:refresh');
```

Comment:
Imagine if you execute this on the last test for the first test class, the migration will be refresh, and then you have second test class that expect migration to not be executed, but due to this the db contain migration from the previous test.